encoding/_yaml/dumper/dumper.ts

```diff
@@ -872,7 +872,7 @@ function getDuplicateReferences(
   for (let index = 0; index < length; index += 1) {
     state.duplicates.push(objects[duplicatesIndexes[index]]);
   }
-  state.usedDuplicates = new Array(length);
+  state.usedDuplicates = Array.from({ length: length });
```

Comment:
Should probably be

```suggestion
  state.usedDuplicates = Array.from({ length });
```

to be consistent